c2/c2-protocol/c2-protocol-api/src/main/java/org/apache/nifi/c2/protocol/api/NetworkInfo.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Apache NiFi - MiNiFi
+ * Copyright 2014-2018 The Apache Software Foundation
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.nifi.c2.protocol.api;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.io.Serializable;
+import javax.validation.constraints.Size;
+
+@ApiModel
+public class NetworkInfo implements Serializable {
+    private static final long serialVersionUID = 6574362622904811901L;
+
+    public static final int DEVICE_ID_SIZE = 200;
+    public static final int IP_ADDRESS_MAX_SIZE = 45;
```

Comment:
Is there a particular reason for making these public properties?  If they are intended to be reusable, it seems better to define them using an `enum`, although a static value would still be necessary for the annotation definition.

Reply:
Nope, just forethought in case they're needed, I can make them private for now if they're unused elsewhere.